client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -252,6 +253,15 @@ public final ClusterClient cluster() {
         return clusterClient;
     }
 
+    /**
+     * Provides a {@link ModulesClient} which can be used to access the various Modules APIs.
+     *
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/modules.html">Modules APIs on elastic.co</a>
+     */
+    public final ModulesClient modules() {
```

Comment:
On the same line as my other comment above, I assume that this method is weird for users. Also, none of the other language clients have this namespace while they all have `indices`, `cluster` etc.